iot-hub/Quickstarts/read-d2c-messages/read_device_to_cloud_messages_async.py

```diff
@@ -0,0 +1,83 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+This sample demonstrates how to use the Microsoft Azure Event Hubs Client for Python async API to 
+read messages sent from a device. Please see the documentation for @azure/event-hubs package
+for more details at https://pypi.org/project/azure-eventhub/
+
+For an example that uses checkpointing, follow up this sample with the sample in the 
+azure-eventhub-checkpointstoreblob package on GitHub at the following link:
+
+https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/samples/receive_events_using_checkpoint_store_async.py
+"""
+
+
+import asyncio
+from azure.eventhub import TransportType
+from azure.eventhub.aio import EventHubConsumerClient
+
+
+# Event Hub-compatible endpoint
+# az iot hub show --query properties.eventHubEndpoints.events.endpoint --name {your IoT Hub name}
+EVENTHUB_COMPATIBLE_ENDPOINT = "{your Event Hubs compatible endpoint}"
+EVENTHUB_COMPATIBLE_ENDPOINT = "iothub-ns-yijun-ioth-3130889-4398262da5.servicebus.windows.net"
+
+# Event Hub-compatible name
+# az iot hub show --query properties.eventHubEndpoints.events.path --name {your IoT Hub name}
+EVENTHUB_COMPATIBLE_PATH = "{your Event Hubs compatible name}"
+EVENTHUB_COMPATIBLE_PATH = "yijun-iothub"
+
+# Primary key for the "service" policy to read messages
+# az iot hub policy show --name service --query primaryKey --hub-name {your IoT Hub name}
+IOTHUB_SAS_KEY = "{your service primary key}"
+IOTHUB_SAS_KEY = "3FQcB8ptGd3QSgcq7vmNwCnRwqCZA4s+Wd2zziBKjaw="
+
+# If you have access to the Event Hub-compatible connection string from the Azure portal, then
+# you can skip the Azure CLI commands above, and assign the connection string directly here.
+CONNECTION_STR = f'Endpoint=sb://{EVENTHUB_COMPATIBLE_ENDPOINT}/;SharedAccessKeyName=service;SharedAccessKey={IOTHUB_SAS_KEY};EntityPath={EVENTHUB_COMPATIBLE_PATH}';
```

Comment:
You don't need the leading 'sb://' here. It's inlcuded in the EVENTHUB_COMPATIBLE_ENDPOINT. Also, I don't think you need/want the terminating semi-colon. Same things for the sync sample

Reply:
Sorry for the trailing ';'. I used VS code to create this sample. I realized it didn't save automatically like Pycharm so my last small fixes were not saved and pushed.
Corrected now.